charters/rust-core/README.md

```diff
@@ -0,0 +1,277 @@
+
+The Rust Core Working Group will make an intentional effort to prioritize matters of security and performance. Furthermore, with regards to API design & developer experience (DX), this Working Group will prioritize first and foremost modularity and extensibility, over DX convenience, in order to limit the liklihood of introducing breaking changes to consumers of the delivered SDKs.
+
+The Rust Core Working Group is not concerned with the definitions of the web5 and tbdex specifications, both of which are the concerns of the respective Working Group. However, the Rust Core Working Group will offer valuable concrete feedback to the web5 and tbdex Working Groups. Although the web5 and tbdex specifications are not within the scope of this Working Group, a considerable amount of cross-Working Group collaboration is to be expected (see [Coordination](#coordination) and [Communication](#communication) below).
+
+The Rust Core Working Group codifies a standard API Design ([web5 APID](#deliverable-1-web5-api-design-apid) & [tbdex APID](#deliverable-6-tbdex-api-design-apid)) which is implemented across all supported language implementations (Rust, Kotlin and Swift), but any additional API features or design characteristics are not within the scope of this Working Group. Each implementation must support the standard API Design as a baseline implementation, but does not exclude the possibility of additional features.
+
+The Rust Core Working Group will codify itself across the following GitHub repos:
+
+- web5: https://github.com/TBD54566975/web5-rs/
+  - [[Deliverable 1]: web5 API Design (APID)](#deliverable-1-web5-api-design-apid)
+  - [[Deliverable 2]: web5 Rust Implementation](#deliverable-2-web5-rust-implementation)
+  - [[Deliverable 3]: web5 UniFFI Binding](#deliverable-3-web5-uniffi-binding)
+  - [[Deliverable 4]: web5 Kotlin Implementation](#deliverable-4-web5-kotlin-implementation)
+  - [[Deliverable 5]: web5 Swift Implementation](#deliverable-5-web5-swift-implementation)
+  - RFCs for historical tracking of significant decisions
+- tbdex: https://github.com/TBD54566975/tbdex-rs/
+  - [[Deliverable 6]: tbdex API Design (APID)](#deliverable-6-tbdex-api-design-apid)
+  - [[Deliverable 7]: tbdex Rust Implementation](#deliverable-7-tbdex-rust-implementation)
+  - [[Deliverable 8]: tbdex UniFFI Binding](#deliverable-8-tbdex-uniffi-binding)
+  - [[Deliverable 9]: tbdex Kotlin Implementation](#deliverable-9-tbdex-kotlin-implementation)
+  - [[Deliverable 10]: tbdex Swift Implementation](#deliverable-10-tbdex-swift-implementation)
+  - RFCs for historical tracking of significant decisions
```

Comment:
```suggestion
- Web5: https://github.com/TBD54566975/web5-rs/
  - [[Deliverable 1]: Web5 API Design (APID)](#deliverable-1-web5-api-design-apid)
  - [[Deliverable 2]: Web5 Rust Implementation](#deliverable-2-web5-rust-implementation)
  - [[Deliverable 3]: Web5 UniFFI Binding](#deliverable-3-web5-uniffi-binding)
  - [[Deliverable 4]: Web5 Kotlin Implementation](#deliverable-4-web5-kotlin-implementation)
  - [[Deliverable 5]: Web5 Swift Implementation](#deliverable-5-web5-swift-implementation)
  - RFCs for historical tracking of significant decisions
- tbDEX: https://github.com/TBD54566975/tbdex-rs/
  - [[Deliverable 6]: tbDEX API Design (APID)](#deliverable-6-tbdex-api-design-apid)
  - [[Deliverable 7]: tbDEX Rust Implementation](#deliverable-7-tbdex-rust-implementation)
  - [[Deliverable 8]: tbDEX UniFFI Binding](#deliverable-8-tbdex-uniffi-binding)
  - [[Deliverable 9]: tbDEX Kotlin Implementation](#deliverable-9-tbdex-kotlin-implementation)
  - [[Deliverable 10]: tbDEX Swift Implementation](#deliverable-10-tbdex-swift-implementation)
  - RFCs for historical tracking of significant decisions
```

Reply:
I was hoping someone would chime in with casing 😃